src/AdminConsole/Pages/Organization/Join.cshtml

```diff
@@ -56,7 +56,6 @@ else
                 </div>
 
                 <input type="hidden" asp-for="Form.Code"/>
-                <div asp-validation-summary="All"></div>
```

Comment:
They appear right under the input fields themselves. So entering ' ' (whitespace) in the 'Name' field for example, will upon post show a red border around the text field with the error message.

If the summary is not removed, then all validation errors would show twice, once under the field, once in the summary list at the bottom of the form.

Reply:
@jonashendrickx While that is true for `Name`, `Code`, `Email`, it's not true for `Form.AcceptsTermsAndPrivacy` because that checkbox don't have a `<span asp-validation-for="xxx"></span>`-element. The validation errors is also not added as a "field-error", but as a `ModelError` on Line 72/63.

If validation-summary is removed, we would also not be able to see the `Invite is invalid or expired` error, right?

I'd recommend changing the summary to a `ModelOnly`, that way we don't repeat field values, but can still display ModelErrors.

Reply:
I've put it back as ModelOnly. But if the checkbox is not checked, it already gives you a small tooltip saying it's mandatory to accept the terms and conditions.